templates/release_checklist.md

```diff
@@ -0,0 +1,149 @@
+- [ ] [Perform general writing flow tests on iOS](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+
+<!-- /scheduled_release_only -->
+
+## Create the Release
+
+<!-- scheduled_release_only -->
+
+On {{release_date}}:
+
+<!-- /scheduled_release_only -->
+
+- [ ] [Verify Aztec is up to date](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md#handling-aztec-updates).
+- [ ] Run `./release_automation.sh` from the release toolkit to generate the main apps integration PRs [\*](#).
+- [ ] Notify team that the release has been cut.
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+⚠️ The gutenberg-mobile {{version_number}} release branches are now cut.
+Please do not merge any Gutenberg-related changes into the WPiOS or WPAndroid
+develop branches until after the main apps cut their own releases next week.
+If you’d like to merge changes now, merge them into the `gutenberg/after_{{version_number}}` branches.
+```
+
+</p>
+</details>
+
+- [ ] [Verify localization string files have been generated properly](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] Update `Unreleased` section in [`RELEASE-NOTES.txt`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) and [`gutenberg/../CHANGELOG.md`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/CHANGELOG.md). Add a new `Unreleased` section after updating the release notes.
+- [ ] Perform quick smoke test of the editor on both iOS and Android builds
+- [ ] Update [Gutenberg Mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pulls?q=is%3Apr+is%3Aopen+label%3Arelease-process) description. Make sure the "Changes" section informs testers on what is in the release.
+- [ ] Mark all 4 PRs ready for review.
+
+<!-- non_scheduled_release_only -->
+
+- [ ] Notify the team members directly resposible for handling the releases of WPiOS and WPAndroid.
+
+<!-- /non_scheduled_release_only -->
+
+<!-- optional_aztec_release -->
+
+<!-- optional_incoming_changes -->
+
+## Integrate the Release
+
+<!-- scheduled_release_only -->
+
+On {{release_date}}:
+
+<!-- /scheduled_release_only -->
+
+- [ ] Verify `gutenberg` ref within the gutenberg-mobile release branch is pointing to the [latest commit](#) in Gutenberg release branch.
+- [ ] Create and push a `rnmobile/{{version_number}}` git tag for the head of [Gutenberg release branch](https://github.com/WordPress/gutenberg/branches/all?query=rnmobile%2Frelease).
+- [ ] [Update bundles in the Gutenberg Mobile release branch](#).
+- [ ] Merge **gutenberg-mobile** PR to `trunk` [using the "Create a merge commit" option](#). **WARNING** do **not** merge gutenberg PR to `trunk` yet.
+- [ ] [Create a new Gutenberg moble release](#).
+- [ ] [Update Gutenberg mobile references in the main apps](#).
```

Comment:
I'm wondering if you were considering adding a section in the release documentation for this, if not, I'd consider adding the following steps here:

> In WPiOS, update the reference to point to the tag of the Release created in the previous task.

> In WPAndroid, update the `gutenbergMobileVersion` in `build.gradle` to point to the tag of the Release used in the previous task.

Reply:
I was going to add this extra info to the release documentation. I prefer to have the checklist item as a reminder and  the details available via a link if some needs to learn or be reminded on how to update the main apps.

Reply:
Ok, I think it's a great idea to include the details of these steps in the release documentation, the clearer the steps are the better to avoid mistakes 😅 .

> I was going to add this extra info to the release documentation.
Are you planning to add them in this PR or we should remove the link in the meantime?